frame/staking/src/lib.rs

```diff
@@ -2247,109 +2280,134 @@ impl<T: Config> Module<T> {
 		}
 	}
 
-	/// Plan a new era. Return the potential new staking set.
-	fn new_era(start_session_index: SessionIndex) -> Option<Vec<T::AccountId>> {
+	/// Plan a new era.
+	///
+	/// * Bump the current era storage (which holds the latest planned era).
+	/// * Store start session index for the new planned era.
+	/// * Clean old era information.
+	/// * Store staking information for the new planned era
+	///
+	/// Returns the new validator set.
+	pub fn new_era(
```

Comment:
I think this should be called `trigger_new_era`.

So the call path would be: 

- `try_trigger_new_era`, and if it works, then -> `trigger_new_era `.

Reply:
yes done